test/end-to-end/cases-coverage/xsl-variable-02-no-opt.xspec

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xspec-test enable-coverage?>
+<?xspec-test saxon-custom-options=-opt:0?>
```

Comment:
With Saxon-HE 12.7, I got the same trace with `-opt:0` or `-opt:-mv`. I don't see a lot of benefit of adding another variant of `xsl-variable-02.xspec` besides this one.